src/main/java/teammates/storage/entity/StudentProfile.java

```diff
@@ -34,8 +36,7 @@
 
     private String nationality;
 
-    /* only accepts "male", "female" or "other" */
```

Comment:
This is not back-compatible. Maybe you can just leave entity as there and change `StudentProfileAttribute` only.